src/umpire/CMakeLists.txt

```diff
@@ -114,6 +136,7 @@ endif ()
 
 blt_list_append( TO umpire_depends ELEMENTS umpire_interface IF ENABLE_C )
 blt_list_append( TO umpire_depends ELEMENTS cuda IF ENABLE_CUDA )
```

Comment:
I think this line can be removed, since only `umpire_device` needs to depend on CUDA directly.